cypress/e2e/course_board/createEditDeleteSingleColumnBoard.feature

```diff
@@ -81,7 +81,7 @@ Feature: Course Board - To create, edit and delete a single column course board
         Given I am logged in as a '<student>' at '<namespace>'
         When I go to courses overview
         When I go to course '<course_name>'
-        Then I can see board on course page
+        Then I can see board on course page '<board_title>'
```

Comment:
sentence structure seems a bit weird to me, maybe `board <board_title> on course page`?

Reply:
changed it.